### PR TITLE
WIP Feature BL 5773 all email addresses lower case

### PR DESCRIPTION
**Pull request by sjduffey**

### CODE REVIEW

File: webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//            return value;
+//        }
+//    }
+
+    private String value;
+
+    private Subscription.ContactType contactType;
+
+    public ContactDetail(String value, Subscription.ContactType contactType) {
+        this.value = value.toLowerCase();
```
Comment (raffcecco) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java]: You weren't to know about his, but the coding convention for MOTR is to have a blank line between method declarations and the method body. As this is a public repo, the team were a bit pedantic about this kind of thing (as I discovered after my first push :) ).

File: webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package uk.gov.dvsa.motr.web.component.subscription.model;
+
+public class ContactDetail {
+
```
Comment (raffcecco) [inline webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java]: Is this class also going to be used in the Subscription Loader and Notifier? Would be nice to get everything symmetrical but maybe outside the scope of the ticket.

File: webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java (hunk 1 above)
Comment (sjduffey) [reply webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java]: Thanks, I've added some blank lines :-)

File: webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java (hunk 2 above)
Comment (sjduffey) [reply webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java]: Uh... not sure. I hadn't considered it at all, I wasn't sure how much overlap there is

Comment (djeley) [reply webapp/src/main/java/uk/gov/dvsa/motr/web/component/subscription/model/ContactDetail.java]: Would be nice, and shouldn't be _too_ much work. Loader and Notifier have copies (almost) of the Subscription class so would be nice to improve consistency across all components.

**Pull request merged**